boost-common/src/main/java/io/openliberty/boost/common/boosters/CDIBoosterConfig.java

```diff
@@ -25,25 +25,8 @@
 @BoosterCoordinates(AbstractBoosterConfig.BOOSTERS_GROUP_ID + ":cdi")
 public class CDIBoosterConfig extends AbstractBoosterConfig {
 
-    String libertyFeature = null;
-
     public CDIBoosterConfig(Map<String, String> dependencies, BoostLoggerI logger) throws BoostException {
```

Comment:
Understand if we're treating this as a "work in progress", not wanting to tear away the reflection calls using this.   Otherwise the **AbstractBoosterConfig** approach obviously no longer seems useful.